cloud/scope/powervs_cluster_test.go

```diff
@@ -1704,3 +1704,515 @@ func TestCreateServiceInstance(t *testing.T) {
 		g.Expect(err).To(BeNil())
 	})
 }
+
+func TestReconcileVPC(t *testing.T) {
+	var (
+		mockVPC  *mock.MockVpc
+		mockCtrl *gomock.Controller
+	)
+
+	setup := func(t *testing.T) {
+		t.Helper()
+		mockCtrl = gomock.NewController(t)
+		mockVPC = mock.NewMockVpc(mockCtrl)
+	}
+	teardown := func() {
+		mockCtrl.Finish()
+	}
+	t.Run("When VPC already exists in cloud, ", func(t *testing.T) {
+		g := NewWithT(t)
+		setup(t)
+		t.Cleanup(teardown)
+
+		clusterScope := PowerVSClusterScope{
+			IBMVPCClient: mockVPC,
+			IBMPowerVSCluster: &infrav1beta2.IBMPowerVSCluster{Spec: infrav1beta2.IBMPowerVSClusterSpec{
+				VPC:           &infrav1beta2.VPCResourceReference{},
```

Comment:
Is this required to define here?

Reply:
Its not required, removed all such references